### ISSUE TITLE
Accessibility feedback - Staging Review - BAM1 [BDD]

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#12569]
**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Point of Contact

**VFS Point of Contact:** Jennifer (VSA) & Trevor (VSP)

## Description

This ticket houses accessibility issues for the staging review of BAM1 - BDD.

## Test cases

1. Staging link:  
1. 

## Definition of done

1. Review and acknowledge feedback.
1. Document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions.

**Comment by jenstrickland** [comment]
It was great to see this in the process, since I remember the Veteran I spoke with who was so frustrated his answers weren't saved in the PDF form he used online. 

> If you need to take a break from the process, your responses are being saved and will be here when you return to your online application.

👏

**Comment by jenstrickland** [comment]
@luke-gcio I'm not sure if I should ask the designers or the developers about this. This instance of two "Update page" buttons stacked is likely to be confusing to many, and especially those with some cognitive considerations. Is this what the screen should look like? 

![Screen Shot 2020-08-20 at 4 21 41 PM](https://user-images.githubusercontent.com/57469/90821664-43315380-e301-11ea-82f8-62f76a389a53.png)

**Comment by jenstrickland** [comment]
In Firefox + NVDA, a strange thing happens on the landing page with this heading level 4. The underlying code doesn't indicate anything. I edited the HTML to remove the period and lower-cased 'evidence', but it still separated it as two instances of heading level 4.

![Screen Shot 2020-08-20 at 5 19 35 PM](https://user-images.githubusercontent.com/57469/90827231-e4240c80-e309-11ea-9c55-b2a8e12202e3.png)

![Screen Shot 2020-08-20 at 5 23 18 PM](https://user-images.githubusercontent.com/57469/90827240-e7b79380-e309-11ea-806f-8f755ab06140.png)

**Comment by jenstrickland** [comment]
This is more of a QA or Design bug, but in Firefox, the Edit buttons overlap the rule beneath the individual headings inside the Review screen accordions. I'm not sure if I should write a ticket for this, and instead will defer to @CrystabelReiter or @tlei123.

![image (6)](https://user-images.githubusercontent.com/57469/90830013-e177e600-e30e-11ea-82d0-dcee52beff49.png)

**Comment by jenstrickland** [comment]
> * With respect to the NVDA bug, which browser were you using? It seems specific enough it might be an issue for NVAccess, but I'll hold until I hear back from you.

@1Copenut wrt the NVDA bug, is this the one? 

> In Firefox + NVDA, a strange thing happens on the landing page with this heading level 4. The underlying code doesn't indicate anything. I edited the HTML to remove the period and lower-cased 'evidence', but it still separated it as two instances of heading level 4. I suspect this is a quirk of NVDA and opted not to write a ticket for it, because it is so minor.
> 
> ![Screen Shot 2020-08-20 at 5 19 35 PM](https://user-images.githubusercontent.com/57469/90827231-e4240c80-e309-11ea-9c55-b2a8e12202e3.png)
> 
> ![Screen Shot 2020-08-20 at 5 23 18 PM](https://user-images.githubusercontent.com/57469/90827240-e7b79380-e309-11ea-806f-8f755ab06140.png)

**Comment by jenstrickland** [comment]
@luke-gcio The feedback is ready for your team.

**Comment by 1Copenut** [comment]
@jenstrickland Yes, the NVDA bug I was asking about was the one you highlighted where the H4 wasn't reading as expected. I'll try to recreate it and see if it's a quirk, or a bug.
